### ISSUE TITLE
Optimize all SVGs

### ISSUE THREAD

**Issue by aashutoshrathi** [gitcoinco/web#3184]
There are too much of SVGs in our codebase and only a few of them are optimized.
I think all of them should be optimized.

cc: @thelostone-mc

**Comment by thelostone-mc** [comment]
Yeah I've been meaning to tak care of that next week ! We've got few big changes coming in , so will get in post that ^_^
